lib/archive/unpack_unix.go

```diff
@@ -37,6 +37,12 @@ const (
 	// O_TRUNC = truncate file to 0 length if it does exist(overwrite the file)
 	// O_WRONLY = We use this since we do not intend to read, we only need to write.
 	fileWriteFlags = os.O_CREATE | os.O_TRUNC | os.O_WRONLY
+
+	// Location of the unpack binary inside of containers
+	ContainerBinaryPath = "/.tether/unpack"
```

Comment:
Do these need to be exported?

Reply:
Ah, as the code exists currently `ContainerBinaryPath` does, but I can think of a short refactor to address that.